[Magic Firewall] Added Plans page (#3874)
cloudflare/cloudflare-docs: content/magic-firewall/plans.md

---
title: Plans
pcx-content-type: concept
weight: 2
---

# Plans

> If you are a [Magic Transit](/magic-transit/) or [Magic WAN](/magic-wan/) user, you are automatically provided with a standard list of Magic Firewall features. For additional features available for purchase, see the list of advanced features below.

Reviewer: ```suggestion
If you are a [Magic Transit](/magic-transit/) or [Magic WAN](/magic-wan/) user, you are automatically provided with a standard list of Magic Firewall features. For additional features available for purchase, refer to the list of advanced features below.
```

> ### Standard features

Reviewer 2: ```suggestion
## Standard features
```

> - Filtering rules based on protocol, port, IP addresses, packet length and bit field match

Reviewer: ```suggestion
- Filtering rules based on protocol, port, IP addresses, packet length, and bit field match
```

- Fast propagation of rule changes in <500ms
- Single dashboard to manage firewall and network configuration

> - Programmable API for automated deployment and management — compatible with infrastructure-as-code platforms like Terraform

Reviewer: ```suggestion
- Programmable API for automated deployment and management — compatible with infrastructure-as-code platforms like [Terraform](/terraform/)
```

> - Traffic analytics per rule in the dashboard and using the GraphQL API

Reviewer: ```suggestion
- Traffic analytics per rule in the dashboard and using the [GraphQL API](/analytics/graphql-api/)
```

> - Integration with Magic WAN network-as-a-service

Reviewer: ```suggestion
- Integration with [Magic WAN network-as-a-service](/magic-wan/)
```

> - Included DDoS protection with Magic Transit

Reviewer: ```suggestion
- Included DDoS protection with [Magic Transit](/magic-transit/)
```

> ### Advanced features

Reviewer 2: ```suggestion
## Advanced features
```

> 

Reviewer 3: I would suggest mentioning that standard features are also included.

Author: Good callout. Added that info

- Customizable IP lists
- Managed threat intelligence IP lists (Malware, Botnet, Anonymizer)
- Geoblocking based on user location by country
- Packet captures on demand for network troubleshooting
- Protocol validation rules to inspect traffic validity and enforce a positive security model

> - Optional upgrade to full stateful Secure Web Gateway using Cloudflare Zero Trust for outbound Internet traffic

Reviewer: ```suggestion
- Optional upgrade to full stateful Secure Web Gateway using [Cloudflare Zero Trust](/cloudflare-one/) for outbound Internet traffic
```
